doc/concepts/data-concepts/provenance.md

```diff
@@ -0,0 +1,102 @@
+transformation chain or the data itself.
+
+For example, when a bank makes a decision about a mortgage
+application, many factors are taken into consideration, including the
+credit history, annual income, and loan size. This data goes through multiple
+automated steps of analysis with numerous dependencies and decisions made
+along the way. If the final decision does not satisfy the applicant,
+the historical data is the first place to look for proof of authenticity,
+as well as for possible prejudice or model bias against the applicant.
+Data provenance creates a complete audit trail that enables data scientists
+to track the data from its origin through to the final decision and make
+appropriate changes that address issues. With the adoption of General Data
+Protection Regulation (GDPR) compliance requirements, monitoring data lineage
+is becoming a necessity for many organizations that work with sensitive data.
+
+Pachyderm implements provenance for both commits and repositories.
+You can track revisions of the data and
+understand the connection between the data stored in one repository
+and the results in the other
+repository.
+
+Collaboration takes data provenance even further. Provenance enables teams
+of data scientists across the globe to build on each other work, share,
+transform, and update datasets while automatically maintaining a
+complete audit trail so that all results are reproducible.
+
+The following diagram demonstrates how provenance works:
+
+![Provenance example](../../images/provenance.png)
+
+In the diagram above, you can see two input repositories called `parameters`
+and `training-data`. The `training-data` repository continuously collects
+data from an outside source. The training model pipeline combines the
+data from these two repositories, trains many models, and runs tests to select the best
+one.
+
+Provenance helps you to understand how and why the best model was
+selected and enables you to track the origin of the best model.
+In the diagram above, the best model is represented with a purple
+circle. By using provenance, you can find that the best model was
+created from the commit **2** in the `training-data` repository
+and the commit **1** in the `parameters` repository.
+
+## Tracking the Provenance Upstream
+
+Pachyderm provides the `pachctl inspect` command that enables you to track
+the provenance of your commits and learn where the data in the repository
+originates in.
+
+**Example:**
+
+```bash
+$ pachctl inspect commit split@master
+Commit: split@f71e42704b734598a89c02026c8f7d13
+Original Branch: master
+Started: 4 minutes ago
+Finished: 3 minutes ago
+Size: 0B
+Provenance:  __spec__@8c6440f52a2d4aa3980163e25557b4a1 (split)  raw_data@ccf82debb4b94ca3bfe165aca8d517c3 (master)
+```
+
+In the example above, you can see that the latest commit in the master
+branch of the split repository tracks back to the master branch in the
+`raw_data` repository.
```

Comment:
Do we want to also mention the "spec" commit? say:

The "spec" provenance also shows you exactly what version of your code was run on the input commit (`raw_data@ccf82debb4b94ca3bfe165aca8d517c3`) to produce the output commit (`split@f71e42704b734598a89c02026c8f7d13`)